modules/installation-osp-balancing-external-loads.adoc

```diff
@@ -7,6 +7,8 @@
 
 Configure an external load balancer in OpenStack to resolve external networking needs.
 
+// Add content about when and why?
```

Comment:
Are there any good blurbs or resources to add here that clarify when and why you'd want to configure a load balancer? @tomassedovic @iamemilio

Reply:
Additionally, regarding this comment: https://github.com/openshift/openshift-docs/pull/16492#discussion_r324194817

And this one: https://github.com/openshift/openshift-docs/pull/16492#discussion_r324192392

What changes should happen here? 

@iamemilio @tomassedovic

Reply:
Some will already have their own load balancing infrastructure, and will probably want to continue to use it. Other than that, the load balancing infrastructure that we set up on the nodes can't really be scaled out easily, so depending on the scale of their cluster, it might be worth their while. Thats all I can think of